core/arch/aarchxx/mangle.c

```diff
@@ -642,7 +647,6 @@ insert_push_all_registers(dcontext_t *dcontext, clean_call_info_t *cci,
         if (spill)
             PRE(ilist, instr, instr_create_restore_from_tls(dcontext, scratch, slot));
     }
-    ASSERT(ALIGNED(dstack_offs, get_ABI_stack_alignment()));
```

Comment:
This assert now belongs up above?

Reply:
No, I don't think so. I check the alignment after pushing `simd` and right at the end. I don't think there's another good place to do it.